sqlcop/checks/cross_join.py

```diff
@@ -51,6 +52,12 @@ def __call__(self, stmt):
                 if isinstance(tok, sqlparse.sql.IdentifierList):
                     tables += tok.get_identifiers()
 
+                # If we are doing full joins between tables, we should be worried.
+                elif isinstance(tok, sqlparse.sql.Identifier) and (
+                    (stmt[i-2].match(sqlparse.tokens.Keyword, 'FROM')) or (
```

Comment:
Seems like there are too many logic here. Can you point to specific test that verifies just the `(stmt[i-2].match(sqlparse.tokens.Keyword, 'FROM'))` and the specific test that verifies this part of the logic `stmt[i-2].match(sqlparse.tokens.Keyword, 'JOIN'))`?